<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportShapefile</class>
 <widget class="QWidget" name="ImportShapefile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Shapefile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioImportAsIs">
     <property name="text">
      <string>Import file as-is</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">boundaryStyleGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioAddMargin">
     <property name="text">
      <string>Add margin to bounding box</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">boundaryStyleGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="marginGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Margin specification</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="textMarginFraction">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioMarginFraction">
        <property name="text">
         <string>Fraction of data diameter:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">marginStyleGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioManualMargin">
        <property name="text">
         <string>Absolute margin:</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">marginStyleGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="textManualMargin">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;i&gt;all&lt;/i&gt; or &lt;i&gt;left+right, bottom+top&lt;/i&gt; or&lt;br&gt;
 &lt;i&gt;left, right, bottom, top&lt;/i&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioManualBounds">
     <property name="text">
      <string>Specify bounding box coordinates</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">boundaryStyleGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Bounding rectangle specification</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="boundingBoxLabel">
        <property name="text">
         <string>Left, right, bottom, top:</string>
        </property>
        <property name="buddy">
         <cstring>textManualBounds</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="textManualBounds"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioCustomBoundary">
     <property name="text">
      <string>Import boundary from second file</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">boundaryStyleGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customBoundaryGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Bounding polyline filename</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="textCustomBoundaryFileName"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCustomBoundaryFileChooser">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioImportAsIs</tabstop>
  <tabstop>radioAddMargin</tabstop>
  <tabstop>radioManualBounds</tabstop>
  <tabstop>radioCustomBoundary</tabstop>
  <tabstop>radioMarginFraction</tabstop>
  <tabstop>textMarginFraction</tabstop>
  <tabstop>radioManualMargin</tabstop>
  <tabstop>textManualMargin</tabstop>
  <tabstop>textManualBounds</tabstop>
  <tabstop>textCustomBoundaryFileName</tabstop>
  <tabstop>buttonCustomBoundaryFileChooser</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioAddMargin</sender>
   <signal>toggled(bool)</signal>
   <receiver>marginGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCustomBoundary</sender>
   <signal>toggled(bool)</signal>
   <receiver>customBoundaryGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualBounds</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="boundaryStyleGroup"/>
  <buttongroup name="marginStyleGroup"/>
 </buttongroups>
</ui>
